build_grpc_gateway.sh

```diff
@@ -1,4 +1,10 @@
 docker build -t grpc-gateway -f grpc-gateway/Dockerfile .
```

Comment:
Could we just do something like `set -euxo pipefail` at the top to make any failed command fail the script?